maestro-app/src/main/java/bio/overture/maestro/app/infra/config/properties/DefaultApplicationProperties.java

```diff
@@ -40,45 +40,45 @@
 
     final static String MAESTRO_PREFIX = "maestro";
 
-    @Value("${maestro.elasticsearch.cluster_nodes}")
```

Comment:
It's a little confusing to have both @ConfigurationProperties and @Value, isn't it? We can make nested class to wrap those nested values
https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html#boot-features-external-config-typesafe-configuration-properties

Reply:
at the time i didn't want to spend the time building classes just for that, done now